src/App.jsx

```diff
@@ -14,70 +16,61 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood
+          ? `${selectedGood} is selected`
+          : 'No goods selected'
+        }
+        {selectedGood && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => setSelectedGood('')}
+          />
+        )}
+      </h1>
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
-
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr
+              data-cy="Good"
+              key={good}
+              className={classNames({
+                'has-background-success-light': selectedGood === good,
+              })}
             >
-              +
-            </button>
-          </td>
+              <td>
+                <button
+                  data-cy={
+                    selectedGood === good
+                      ? 'RemoveButton'
+                      : 'AddButton'
+                  }
+                  type="button"
+                  className={classNames({
+                    'is-info': selectedGood === good,
+                  })}
+                  onClick={() => setSelectedGood(selectedGood === good
+                    ? '' : good)}
+                >
+                  {selectedGood === good ? '-' : '+'}
+                </button>
```

Comment:
I suggest if you change almost everything depending on the selectedGood === good condition, will be better to write
```
{selectedGood === good 
  ? <button></button>
   : <button></button>
}
```

Reply:
I change my code like you said))
Please explaine why it's better?

Reply:
it looks nicer and cleaner when you don't have extra condition inserts

Reply:
<img width="1455" alt="image" src="https://github.com/mate-academy/react_goods-selector-js/assets/70220667/f29d04fd-81f7-4de5-bc85-3d5679f976b9">
compare yourself, which is better read

Reply:
Yes, you right)
Thank's)